apps/src/lib/client/tx.rs

```diff
@@ -420,6 +426,81 @@ pub async fn submit_transfer(ctx: Context, args: args::TxTransfer) {
     process_tx(ctx, &args.tx, tx, &keypair).await;
 }
 
+pub async fn submit_create_nft(ctx: Context, args: args::NftCreate) {
+    let file = File::open(&args.nft_data).expect("File must exist.");
+    let nft: Nft = serde_json::from_reader(file)
+        .expect("Couldn't deserialize nft data file");
+
+    let vp_code = match &nft.vp_path {
+        Some(path) => {
+            std::fs::read(path).expect("Expected a file at given code path")
+        }
+        None => ctx.read_wasm(VP_NFT),
+    };
+
+    let data = CreateNft {
+        tag: nft.tag.to_string(),
+        creator: nft.creator,
+        vp_code,
+        keys: nft.keys,
+        opt_keys: nft.opt_keys,
+        tokens: nft.tokens,
+    };
+
+    let data = data.try_to_vec().expect(
+        "Encoding transfer data to initialize a new account shouldn't fail",
+    );
+
+    let tx_code = std::fs::read(TX_CREATE_NFT)
+        .expect("Expected a file at given code path");
```

Comment:
These have to be read via the wasm loader from ctx:
```suggestion
    let tx_code = ctx.read_wasm(TX_CREATE_NFT);
```